pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -104,6 +104,7 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 			Rules: []authorizationapi.PolicyRule{
 				{Verbs: util.NewStringSet("get"), Resources: util.NewStringSet("users"), ResourceNames: util.NewStringSet("~")},
 				{Verbs: util.NewStringSet("list"), Resources: util.NewStringSet("projectrequests")},
+				{Verbs: util.NewStringSet("list", "get"), Resources: util.NewStringSet("clusterroles")},
```

Comment:
Why would they?  I can see a need for that when you're trying to correctly map your users to roles (the admin doing the binding).